tests/functional/aws-node-sdk/lib/utility/versioning-util.js

```diff
@@ -28,15 +28,14 @@ function _deleteVersionList(versionList, bucket, callback) {
     return s3Client.send(new DeleteObjectsCommand(params)).then(() => callback()).catch(err => callback(err));
 }
 
-function checkOneVersion(s3, bucket, versionId, callback) {
+function checkOneVersion(s3, bucket, versionId) {
     return s3Client.send(new ListObjectVersionsCommand({ Bucket: bucket })).then(data => {
```

Comment:
```suggestion
async function checkOneVersion(s3, bucket, versionId) {
return await s3Client.send(new ListObjectVersionsCommand({ Bucket: bucket })).then(data => {
```

This will keep the `checkOneVersion` in the stack trace. It's a "good" practice for async/await. But nits